internal/policies/client.go

```diff
@@ -229,11 +236,12 @@ func (f *Client) groupPoliciesByGVR(ctx context.Context, policies []policiesv1.P
 
 		url, err := f.getPolicyServerURLRunningPolicy(ctx, policy)
 		if err != nil {
-			return nil, err
+			erroredPolicies[policy.GetUniqueName()] = struct{}{}
+			log.Error().Err(err).Str("policy", policy.GetUniqueName()).Msg("failed to obtain matching policy-server URL, skipping as error...")
+			continue
```

Comment:
Shouldn't these belong to "skipped"? They are properly configured (meaning they reference valid GVR), but they are not yet active

Reply:
+1, I think these should go to skipped

Reply:
No, in this case, we already know the policy is active from the previous [if block](https://github.com/kubewarden/audit-scanner/blob/main/internal/policies/client.go#L223-L228), and we are failing to construct a valid policy-server URL.

Reply:
You're right, leave them as errored. This is really a corner case